core/config/initializers/00_configuration.rb

```diff
@@ -131,16 +131,6 @@
         in the default config above.
       ).squish
 
-    field 'Search Facet Default Sort',
```

Comment:
Why are we dropping this? Seemed useful.

Reply:
The default sort only applies to facets that do that have a sort specified. This caused confusion for Jeff, and would likely for any admin as the facet-specific sorts are not admin configurable due to procs and classes being allowed as options. Given you can't see or change the specific ones, the generic default seems to do nothing out of the box as it does not affect the default facet types, which all have a sort specified in the static config.